pages/Favorites.js

```diff
@@ -1,15 +1,18 @@
 import Link from "next/link";
+import Layout from "@/components/Layout";
 
 export default function Favorites() {
   return (
-    <>
-      <h1>Places 2 Visit</h1>
-      <ul>
-        <li>test</li>
-        <li>test</li>
-        <li>test</li>
-        <li>test</li>
-      </ul>
-    </>
+    <Layout>
+      <>
+        <h1>Places 2 Visit</h1>
+        <ul>
+          <li>test</li>
+          <li>test</li>
+          <li>test</li>
+          <li>test</li>
+        </ul>
+      </>
+    </Layout>
```

Comment:
It's not particularly valuable to show a list with dummy text. As long as a feature is not yet completed, leave it out.

Reply:
maybe it is not advantageous that i created the navbar first. can i still leave the navbar but remove /favorites?